app/UI/app.tsx

```diff
@@ -475,17 +480,21 @@ export function getFocusedContent(
 }
 
 async function sync_dm_events(
-    database: Database_View,
     ctx: NostrAccountContext,
-    pool: ConnectionPool,
+    args: {
+        database: Database_View;
+        pool: ConnectionPool;
+        dmController: DirectedMessageController;
```

Comment:
It's better to pass in a smaller data type such as the timestamp itself

Reply:
ok